tools/dotnet-scaffold-aspire/Commands/CachingCommand.cs

```diff
@@ -0,0 +1,111 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Diagnostics.CodeAnalysis;
+using System.Linq;
+using Microsoft.DotNet.Scaffolding.Helpers.General;
+using Microsoft.DotNet.Scaffolding.Helpers.Services;
+using Microsoft.DotNet.Tools.Scaffold.Aspire.Helpers;
+using Spectre.Console;
+using Spectre.Console.Cli;
+
+namespace Microsoft.DotNet.Tools.Scaffold.Aspire.Commands
+{
+    public class CachingCommand : Command<CachingCommand.CachingCommandSettings>
+    {
+        private readonly ILogger _logger;
+        private readonly IFileSystem _fileSystem;
+        private readonly IAppSettings _appSettings;
+        public CachingCommand(IFileSystem fileSystem, ILogger logger, IAppSettings appSettings)
+        {
+            _appSettings = appSettings;
+            _fileSystem = fileSystem;
+            _logger = logger;
+        }
+
+        public override int Execute([NotNull] CommandContext context, [NotNull] CachingCommandSettings settings)
+        {
+            Debugger.Launch();
```

Comment:
Helpful for my debugging purposes but I assume you don't mean to leave this in here :)

Reply:
oops will remove tyty